hedera-node/test-clients/src/main/java/com/hedera/services/bdd/spec/transactions/HapiTxnOp.java

```diff
@@ -884,6 +884,22 @@ public void updateStateFromRecord(TransactionRecord record, HapiSpec spec) throw
         updateStateOf(spec);
     }
 
+    /**
+     * Since we are not submitting inner transactions, we need to set them as submitted
```

Comment:
I don't understand, how are we testing batch transactions at all if inner transactions aren't submitted to the target test network?